app/helpers/view_helper.rb

```diff
@@ -65,16 +65,29 @@ def provider_enrichment_error_url(provider:, field:)
     }[field]
   end
 
-  def header_environment_class
-    "app-header--#{Settings.environment.selector_name}"
+  def environment_colour
+    return "purple" if sandbox_mode?
+
+    case Settings.environment.selector_name
+    when "qa"
+      "orange"
+    when "staging"
+      "red"
+    when "development"
+      "grey"
+    when "review"
+      "purple"
+    when "unknown-environment"
```

Comment:
Can we have a pretty rollover one please 🙏

Reply:
Done. How’s about this rather fetching turquoise?

<img width="1004" alt="Screenshot 2021-05-24 at 21 22 28" src="https://user-images.githubusercontent.com/813383/119403478-54672080-bcd6-11eb-82a1-1cd9194df60b.png">

Reply:
Love it